onprem-environment

```diff
@@ -15,4 +15,8 @@ AGENT_DOCKER_IMAGE=local.registry:5001/ocpmetal/agent:latest
 CONNECTIVITY_CHECK_IMAGE=local.registry:5001/ocpmetal/connectivity_check:latest
 INVENTORY_IMAGE=local.registry:5001/ocpmetal/inventory:latest
 HARDWARE_INFO_IMAGE=local.registry:5001/ocpmetal/hardware_info:latest
-FREE_ADDRESSES_IMAGE=local.registry:5001/ocpmetal/free_addresses:latest
\ No newline at end of file
+FREE_ADDRESSES_IMAGE=local.registry:5001/ocpmetal/free_addresses:latest
+IMAGE_CONTENT_MIRROR=local.registry:5000/ocp4
+IMAGE_CONTENT_SOURCE_1=quay.io/openshift-release-dev/ocp-release
+MIRROR_SOURCE_IMAGE_2=quay.io/openshift-release-dev/ocp-v4.0-art-dev
+ADDITIONAL_TRUST_BUNDLE=<YOUR_REGISTRY_CA_CERT_HERE>
```

Comment:
What format should this be provided?

Reply:
How about this? `YOUR_REGISTRY_CA_CERT_MINIFIED_STRING_HERE`

Reply:
Have you tried testing it in this format? Do we need to minify here? 
ADDITIONAL_TRUST_BUNDLE="BEGIN CERTIFICATE-----
    .....................
    END CERTIFICATE-----"

Reply:
The trust bundle ca-cert needs to be a minified string else podman fails to start the installer container as the `--env-file` flag can't parse multiline value trust bundle cert.